components/Registration/Mitra/index.vue

```diff
@@ -393,6 +416,47 @@ export default {
       } else {
         this.isDropdownOpen = false
       }
+    },
+    fetchUser (token) {
+      try {
+        this.$axios.get('https://www.googleapis.com/oauth2/v2/userinfo', {
```

Comment:
discuss:
should this url saved on env or not? wdyt

Reply:
I agree to save this URL in env kak @yoslie